tests/zfs-tests/tests/functional/zvol/zvol_misc/zvol_misc_common.kshlib

```diff
@@ -96,7 +74,7 @@ function blockdev_missing # device
 	# because there are other commands (zfs snap, zfs inherit, zfs destroy)
 	# that can affect device nodes
 	for i in {1..3}; do
-		udev_wait
+		block_device_wait
```

Comment:
Can we pass the device here to make it a bit more robust?

Reply:
No, we're waiting for the device to go away here so that wouldn't behave correctly. We'd end up not waiting while it's there and then waiting when it goes away.